src/app/src/main/kotlin/org/vaccineimpact/api/app/controllers/UserController.kt

```diff
@@ -0,0 +1,26 @@
+package org.vaccineimpact.api.app.controllers
+
+import org.vaccineimpact.api.app.ActionContext
+import org.vaccineimpact.api.app.controllers.endpoints.SecuredEndpoint
+import org.vaccineimpact.api.app.errors.UnknownObjectError
+import org.vaccineimpact.api.models.User
+
+class UserController(context: ControllerContext) : AbstractController(context)
+{
+    override val urlComponent = "/users"
+    override val endpoints = listOf(
+            SecuredEndpoint("/:username/", this::getUser, emptySet())
+    )
+
+    fun getUser(context: ActionContext): User
+    {
+        var userName = userName(context)
+        var user = repos.user().use { it.getUserByUsername(userName(context))  }
```

Comment:
I assume you meant to use the userName variable here, rather than invoking the userName method again